### ISSUE TITLE
[macOS] Version 0.96.4 can't be launched.

### ISSUE THREAD

**Issue by superheher** [clangen/musikcube#409]
Hello, I am trying your app for the first time, but faced a problem right away. =(
This is a log from the terminal.
<details>

```
dyld: Library not loaded: @rpath/libmusikcore.dylib
  Referenced from: /Users/user/Downloads/musikcube_macos_0.96.4/./musikcube
  Reason: no suitable image found.  Did find:
	/Users/user/Downloads/musikcube_macos_0.96.4/libmusikcore.dylib: code signature in (/Users/user/Downloads/musikcube_macos_0.96.4/libmusikcore.dylib) not valid for use in process using Library Validation: library load disallowed by system policy
[1]    47648 abort      ./musikcube
```
</details>

In the same time `0.96.3` version is going good.

OS: macOS Catalina 10.15.7.

**Comment by clangen** [comment]
Thanks for the heads up, I'm pretty sure I know what the problem is. I'll get some new builds prepared here soon.

**Comment by clangen** [comment]
Published 0.96.5 -- can you let me know if this solves your issue? https://github.com/clangen/musikcube/releases/tag/0.96.5

**Comment by superheher** [comment]
Yeah, you have fixed my problem. Thanks for your fast reply.
